x-pack/legacy/plugins/siem/public/components/embeddables/map_tool_tip/map_tool_tip.tsx

```diff
@@ -67,24 +66,12 @@ export const MapToolTip = React.memo<MapToolTipProps>(
               getLayerName(layerId),
             ]);
 
-            // Fetch ES filters in advance while loader is present to prevent lag when user clicks to add filter
-            const featurePropsPromises = await Promise.all(
-              featureProperties.map(property => property.getESFilters())
-            );
-            const featurePropsESFilters = featureProperties.reduce(
-              (acc, property, index) => ({
-                ...acc,
-                [property._propertyKey]: featurePropsPromises[index],
-              }),
-              {}
-            );
-
             setFeatureProps(featureProperties);
-            setFeaturePropsFilters(featurePropsESFilters);
             setFeatureGeometry(featureGeo);
             setLayerName(layerNameString);
           } catch (e) {
             setIsError(true);
+            throw e;
```

Comment:
are you sure that you want to add this `throw`? I always worry when I am seeing a throw on the UI